hq/conf/routes

```diff
@@ -7,6 +7,7 @@ GET        /                                          controllers.HQController.i
 
 GET        /iam                                       controllers.HQController.iam
 GET        /iam/:accountId                            controllers.HQController.iamAccount(accountId)
+GET        /iam/:accountId/cred-report                controllers.HQController.generateCredentialsReport(accountId)
```

Comment:
Is there a reason to put this on a separate path? It feels like this is exactly what people will want to see when they go to `/iam/:accountId`.

Reply:
nope, just keeping it separate while testing.